packages/kbn-logging/src/ecs/event.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * @internal
+ */
+export interface EcsEvent {
+  action?: string;
+  category?: EcsEventCategory[];
+  code?: string;
+  created?: string;
+  dataset?: string;
+  duration?: number;
+  end?: string;
+  hash?: string;
+  id?: string;
+  ingested?: string;
+  kind?: EcsEventKind;
+  module?: string;
+  original?: string;
+  outcome?: EcsEventOutcome;
+  provider?: string;
+  reason?: string;
+  reference?: string;
+  risk_score?: number;
+  risk_score_norm?: number;
+  sequence?: number;
+  severity?: number;
+  start?: string;
+  timezone?: string;
+  type?: EcsEventType[];
+  url?: string;
+}
+
+/**
+ * @public
+ */
+export type EcsEventCategory =
```

Comment:
Previously we had all of these types as enums, which I think is often a nicer DX, however I switched to a list of strings here because:
1. I wanted to minimize the amount of downstream code that required a new import (turns out there wasn't much anyway)
2. Long term if ECS starts providing generated types, they will most likely not be using more complex implementations like custom enums. So my goal was to write all of the ECS types in a simple enough way that we could potentially switch over to generated types in the future if we wanted to. (This is also the reason why I avoided any fancy generics in the ECS typedefs themselves).